red.js

```diff
@@ -166,7 +166,7 @@ try {
 }
 
 if (settings.httpAdminRoot !== false && settings.httpAdminAuth) {
-    RED.log.warn("use of httpAdminAuth is deprecated. Use adminAuth instead");
+    RED.log.warn(log._("httpadminauth-deprecated"));
```

Comment:
`log` isn't defined in this file, you have to access it as `RED.log`.

However, the entire i18n system isn't initialised until the call to `RED.start()` on line 215. So none of the messages before this line will resolve.

Don't have a solution for that right now. Will think on it over the weekend.